content/docs/command-reference/remote/modify.md

```diff
@@ -406,7 +406,42 @@ application. To use a custom authentication method, use the following parameters
   ```dvc
   $ dvc remote modify myremote allow_anonymous_login true
   ```
+**Authentication examples:**
 
+Authentication example with `connection_string`:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify --local myremote connection_string 'mysecret'
+$ dvc remote push
+```
+  
+Authentication example with currently logged in az cli user/identity:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify myremote account_name 'myaccount'
```

Comment:
This should probably be the first example — if we're keeping these.

Reply:
Yes, this would make sense as a first example. I also think it would make sense to note that this example depends on a logged in `az cli` user who has at least `Storage Blob Data Contributor` role on given storage account. (and/or link to the corresponding azure documentation)